playbooks/gce/openshift-cluster/launch.yml

```diff
@@ -28,33 +28,33 @@
       type: "{{ k8s_type }}"
       g_sub_host_type: "{{ sub_host_type }}"
 
-  - include: ../../common/openshift-cluster/set_node_launch_facts_tasks.yml
-    vars:
-      type: "infra"
-      count: "{{ num_infra }}"
-  - include: tasks/launch_instances.yml
-    vars:
-      instances: "{{ infra_names }}"
-      cluster: "{{ cluster_id }}"
-      type: "{{ k8s_type }}"
-      g_sub_host_type: "{{ sub_host_type }}"
-
-  - set_fact:
-      a_infra: "{{ infra_names[0] }}"
-  - add_host: name={{ a_infra }} groups=service_master
-
+#  - include: ../../common/openshift-cluster/set_infra_launch_facts_tasks.yml
+#    vars:
+#      type: "infra"
+#      count: "{{ num_infra }}"
+#  - include: tasks/launch_instances.yml
+#    vars:
+#      instances: "{{ infra_names }}"
+#      cluster: "{{ cluster_id }}"
+#      type: "{{ k8s_type }}"
+#      g_sub_host_type: "{{ sub_host_type }}"
+#
+#  - set_fact:
+#      a_infra: "{{ infra_names[0] }}"
+#  - add_host: name={{ a_infra }} groups=service_master
+#
```

Comment:
@twiest 


We use `set_node_launch_facts_tasks.yml` for `infra_node` (https://github.com/openshift/openshift-ansible/blob/master/playbooks/gce/openshift-cluster/launch.yml#L31) : that's not working obviously, the playbook stop on error while trying to launch infra nodes instances.

I tried to repair it with `set_infra_launch_facts_tasks.yml`, the instances are correctly launched, however I encountered many issues at that time (in August).

Currently the GCE deployment isn't working at all with or without infra-node. I suppose nobody is working with it. If somebody is working with it, he/she has to tell me how it can work in the current state.

By commenting this code, the rest is working.
We have to wait an other patch in order to make the infra-node work.

This PR fix the standard deployment without infra and will allow people to work on GCE deployment and repair the infra-node